src/pages/List.vue

```diff
@@ -11,6 +11,19 @@ export default {
     components: {
         MonitorList,
     },
+    watch: {
+        "$root.isMobile"(newVal) {
+            if (!newVal && this.$route.path === "/list") {
+                this.$router.push("/dashboard");
```

Comment:
I suggest you tu use `replace` instead of `push`. You'll avoid having the `/list` and the `/dashboard` in the browser history (you'll instead only have `/dashboard`).

`this.$router.replace({ path: "/new_path" })`

Doc: https://router.vuejs.org/guide/essentials/navigation.html#Replace-current-location